AdityaTiwari/TaskNinja20/app/src/main/java/com/example/taskninja20/reusable/CustomTextRegular.kt

```diff
@@ -0,0 +1,104 @@
+// CustomTextComponents.kt
+package com.example.taskninja20.reusable
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.TextUnit
+import androidx.compose.ui.unit.sp
+import com.example.taskninja20.ui.theme.RobotoFontFamily
+
+@Composable
+fun CustomTextRegular(
+    text: String,
+    color: Color = MaterialTheme.colorScheme.onBackground,
+    fontSize: TextUnit = 16.sp,
+    modifier: Modifier = Modifier,
+    fontFamily: FontFamily = RobotoFontFamily
+) {
+    Text(
+        text = text,
+        color = color,
+        fontSize = fontSize,
+        fontWeight = FontWeight.Normal,
+        fontFamily = fontFamily,
+        modifier = modifier
+    )
+}
+
+@Composable
+fun CustomTextBold(
+    text: String,
+    color: Color = Color.Black,
+    fontSize: TextUnit = 16.sp,
+    modifier: Modifier = Modifier,
+    fontFamily: FontFamily = RobotoFontFamily
+) {
+    Text(
+        text = text,
+        color = color,
+        fontSize = fontSize,
+        fontWeight = FontWeight.Bold,
```

Comment:
If fontWeight is the only tihng which is chaing in these function, can you use some better approach? 
Can you have a single text style and apply as per needed?